src/components/common/BreadcrumbCommon/BreadcrumbCommon.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import s from './BreadcrumbCommon.module.scss'
+
+import BreadcrumbItem from './components/BreadcrumbItem/BreadcrumbItem'
+import BreadcrumbSeparator from './components/BreadcrumbSeparator/BreadcrumbSeparator'
+
+interface BreadcrumbCommonProps {
+    crumbs: { link:string, name:string }[];
+    showHomePage?: boolean;
+}
+
+const BreadcrumbCommon = ({ crumbs, showHomePage=true } : BreadcrumbCommonProps) => {
+    if (showHomePage) {
+        crumbs.unshift({link: "/", name: "Home"});
+    }
+    return (
+        <section className={s.breadcrumbCommon}>
+            {
+                crumbs.map((crumb, i) => {
+                    if (i === 0) {
+                        return (
+                            <BreadcrumbItem key={i} text={crumb.name} href={crumb.link} />
+                        )
+                    }
+                    if (i === crumbs.length-1) {
+                        return (
+                            <BreadcrumbSeparator key={i}>
+                                <BreadcrumbItem key={i} text={crumb.name} href="#" />
+                            </BreadcrumbSeparator>
+                        )
+                    }
+                    return (
+                        <BreadcrumbSeparator key={i}>
+                            <BreadcrumbItem key={i} text={crumb.name} href={crumb.link} />
+                        </BreadcrumbSeparator>
+                    )
+                })
```

Comment:
We shouldn't use index as key
Read more here [https://reactjs.org/docs/lists-and-keys.html](url)
```suggestion
                crumbs.map((crumb, i) => {
                    if (i === 0) {
                        return (
                            <BreadcrumbItem key={crumb.name} text={crumb.name} href={crumb.link} />
                        )
                    }
                    if (i === crumbs.length-1) {
                        return (
                            <BreadcrumbSeparator key={crumb.name}>
                                <BreadcrumbItem text={crumb.name} href="#" />
                            </BreadcrumbSeparator>
                        )
                    }
                    return (
                        <BreadcrumbSeparator key={crumb.name}>
                            <BreadcrumbItem text={crumb.name} href={crumb.link} />
                        </BreadcrumbSeparator>
                    )
                })
```

Reply:
The last item is not a link, it is just a string
![image](https://user-images.githubusercontent.com/76729908/132151570-33ceefad-3677-4ff1-864a-e6349082354e.png)